src/lib/makeParser.spec.ts

```diff
@@ -0,0 +1,109 @@
+import makeParser, { ParseError } from './makeParser'
```

Comment:
I have no hard pref, but I tend to use `__test__/*.spec.ts` for the tests. What standard should we use?
with or without `__test__` ?

Reply:
I tend to like to put the spec files directly next to what they're testing, because in general I like to 1. colocate code as much as possible, and 2. use flat structures as much as possible. However, I do think it's just a matter of personal preference — as the project lead, if you'd prefer to use `__test__`, I'll defer to you.